src/features/evaluations/UserEvaluationAdministrationsTable.tsx

```diff
@@ -28,40 +30,43 @@ export const UserEvaluationAdministrationsTable = () => {
 
   return (
     <div className='flex flex-col gap-8'>
-      <table className='w-full table-fixed'>
-        <thead className='text-left border-b-2'>
-          <tr>
-            <th className='pb-2'>Name</th>
-            <th className='pb-2'>Period</th>
-            <th className='pb-2'>Schedule</th>
-            <th className='pb-2'>Total Evaluations</th>
-            <th className='pb-2'>Total Submitted</th>
-            <th className='pb-2'>Total Pending</th>
-          </tr>
-        </thead>
-        <tbody>
-          {user_evaluation_administrations.map((evaluationAdministration) => (
-            <tr
-              className='cursor-pointer hover:bg-slate-100'
-              key={evaluationAdministration.id}
+      {user_evaluation_administrations.map((evaluationAdministration) => (
+        <div
+          key={evaluationAdministration.id}
+          className='flex flex-col items-start gap-4 md:flex-row md:justify-between'
+        >
+          <div className='flex-1'>
+            <Button
+              variant='unstyled'
               onClick={() => handleViewEvaluation(evaluationAdministration.id)}
             >
-              <td className='py-2'>{evaluationAdministration.name}</td>
-              <td className='py-2'>
-                {formatDate(evaluationAdministration.eval_period_start_date)} to{" "}
-                {formatDate(evaluationAdministration.eval_period_end_date)}
-              </td>
-              <td className='py-2'>
-                {formatDate(evaluationAdministration.eval_schedule_start_date)} to{" "}
-                {formatDate(evaluationAdministration.eval_schedule_end_date)}
-              </td>
-              <td className='py-2'>{evaluationAdministration.totalEvaluations}</td>
-              <td className='py-2'>{evaluationAdministration.totalSubmitted}</td>
-              <td className='py-2'>{evaluationAdministration.totalPending}</td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+              <h2 className='text-primary-500 text-lg font-semibold'>
+                {evaluationAdministration.name}
+              </h2>
+            </Button>
+            <p>
+              Evaluation Period:{" "}
+              {convertToMonthDate(evaluationAdministration.eval_period_start_date)} -{" "}
+              {convertToFullDate(evaluationAdministration.eval_period_end_date)}
+            </p>
```

Comment:
Is it possible that the periods will be on different years? If so, only the end date year is displayed here 🤔

Reply:
can you ask sheila about this

Reply:
I think it's the same with this one, but yeah will ask ate Shei muna
![image](https://github.com/nerubia/kh360-react/assets/91468846/a768da66-bbea-4b45-aa9b-c944b31309f0)

Reply:
Just checked with ate Shei, format for different year should be mmm dd, yyyy - mmm dd, yyyy kuys @jatnerubia

Reply:
I mean, the format I pasted here applies to this page as well

Reply:
alright